### PR TITLE
WPF Linker analyzer tab

### LINKED ISSUE #6: WPF Linker Tab component
**Linker functionality should allow the user to find games installed on platforms (ILinkFinder implementations),  such as Microsoft Store, Origin or other installed platforms in the system, and create a link on another platform, allowing to start a game from platforms that not belongs to.**
- **In the UI ,for the MVP:**
  - Allow the user to select one single platform (ILinkAssigner: Steam in the MVP) to create the link
  - Allow the user to select multiple platform for searching games (ILinkFinder: Xbox and Origin in MVP)
  - Display results of the different ILinkFinder's on a list view
  - Allow the user to select multiple games for creating a link on then ILkinkAssigner selected
  - Display the assignation result in the UI
  - A loading must be shown while the system is loading results and also while is creating the link
  - Asynchronous and parallelization tasks must be performed
- **Not now, but future considerations:**
  - _Steam allows to add a badge image for having better UI link in Big Picture and classic views, we can add this information or even more in the future as an enhancement_

### PR DESCRIPTION
**Pull request by vvillena**
New Implementations

- #6 : WPF Linker analyzer tab
  - Use Dropdown and MultiDropdown from CommonNET
  - Use loading from CommonNET

### CODE REVIEW

File: ChustaSoft.GamersPlatformUtils.UI/Modules/Linker/LinkerControlViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace ChustaSoft.GamersPlatformUtils.UI.Modules.Linker
+{
+    public class LinkerControlViewModel : TraceableViewModelBase<LinkerControlModel>
+    {
+        private const string STEAM_PLATFORM_NAME = "steam";
```
Comment (xaberue) [inline ChustaSoft.GamersPlatformUtils.UI/Modules/Linker/LinkerControlViewModel.cs]: Use Steam public constant in it's own layer instead

File: ChustaSoft.GamersPlatformUtils.UI/Modules/Linker/LinkerControlModel.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+using ChustaSoft.GamersPlatformUtils.Abstractions;
+using System.Collections.ObjectModel;
+using System.IO;
+
+namespace ChustaSoft.GamersPlatformUtils.UI.Modules.Linker
+{
+    public class LinkerControlModel : ViewModelBase
+    {
+
+        private bool _isLoading;
```
Comment (xaberue) [inline ChustaSoft.GamersPlatformUtils.UI/Modules/Linker/LinkerControlModel.cs]: IsLoading property is a bindable property highly related to visibility. The ViewModel is the site for this, the Model is more related to pure data, not to properties related to the actions of a control. 

I recommend moving this property to the ViewModel (LinkerControlViewModel)

**Review by xaberue — CHANGES_REQUESTED** [review]

File: ChustaSoft.GamersPlatformUtils.UI/Modules/Linker/LinkerControlViewModel.cs (hunk 1 above)
Comment (vvillena) [reply ChustaSoft.GamersPlatformUtils.UI/Modules/Linker/LinkerControlViewModel.cs]: Fixed

File: ChustaSoft.GamersPlatformUtils.UI/Modules/Linker/LinkerControlModel.cs (hunk 2 above)
Comment (vvillena) [reply ChustaSoft.GamersPlatformUtils.UI/Modules/Linker/LinkerControlModel.cs]: Property moved

**Review by xaberue — APPROVED** [review]
Awesome work bro!!! Thank you!!! And congratulations :)

**Review by Arthus15 — APPROVED** [review]

**Pull request merged**